client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -81,16 +77,22 @@ const Grid = styled.div`
 const Row = styled.div`
 	display: flex;
 	justify-content: space-between;
-	aling-items: center;
+	margin-bottom: -1px;
+
+	&.plan-comparison-grid__group-title-row {
+		cursor: pointer;
+	}
 
 	@media ( min-width: 880px ) {
 		margin: 0 20px;
 		padding: 12px 0;
 		border-bottom: 1px solid #eee;
 
 		&.plan-comparison-grid__group-title-row {
+			cursor: default;
 			border-bottom: none;
 			padding: 20px 0 10px;
+			pointer-events: none;
 		}
 	}
```

Comment:
This sort of CSS grows in complexity. A lot of this complexity can be avoided by moving this part to it's own styled component `<TitleRow />`

Reply:
Good idea. Move this to `TitleRow`